src/js/index.js

```diff
@@ -1,7 +1,389 @@
-const onUserCreateHandler = () => {
+window.onload = () => {
+  getUserList();
+}
+
+const $ = (selector) => document.querySelector(selector)
+
+const apiUrl = 'https://js-todo-list-9ca3a.df.r.appspot.com';
+const userList = $('#user-list');
+const todoList = $('.todo-list');
+const newTodoInput = $('.new-todo');
+const userName = $('#user-title')
+.getElementsByTagName('span')[0]
+.getElementsByTagName('strong')[0];
+const totalCount = $('.todo-count')
+.getElementsByTagName('strong')[0];
```

Comment:
```suggestion
const totalCount = $('.todo-count strong')
```
저는 위와 같은 방식으로 selector를 사용했는데 혹시 이 방법은 어떻게 생각하시나요?